aws/src/integration/java/org/apache/iceberg/aws/TestDefaultAwsClientFactory.java

```diff
@@ -39,7 +40,7 @@ public void testGlueEndpointOverride() {
     properties.put(AwsProperties.GLUE_CATALOG_ENDPOINT, "https://unknown:1234");
     AwsClientFactory factory = AwsClientFactories.from(properties);
     GlueClient glueClient = factory.glue();
-    Assertions.assertThatThrownBy(
+    assertThatThrownBy(
```

Comment:
Can't we avoid these cleanups with this PR? IMO, this PR should have only Migration from Junit4 to Junit5 . WDYT?

Reply:
I believe the changes for removing `Assertion` are very small in this package, and by the changes, it can avoid the situation that two methods like `Assertions.assertThat` and `assertThat` exist in a same file. So it won't be a big problem. Is there any reason to keep `Assertions`?
[Iceberg contribute guide](https://iceberg.apache.org/contribute/#assertj) also shows the tests without `Assertions` .